products/pan-os.md

```diff
@@ -14,7 +14,7 @@ releases:
 -   releaseCycle: "11.0"
     eol: 2024-11-17
     releaseDate: 2022-11-17
-    latest: "11.0.0"
+    latest: "11.0"
```

Comment:
Should be 11.0.0 according to https://docs.paloaltonetworks.com/pan-os/11-0/pan-os-release-notes/pan-os-11-0-0-known-and-addressed-issues/pan-os-11-0-0-addressed-issues.